### PR TITLE
Remove lodash

### PR DESCRIPTION
**Pull request by sharkparty**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [x] Used [conventional commits](https://www.conventionalcoamits.org/) for all work.
- [x] Tested my solution on Mobile & Tablet.
- [x] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [x] Updated snapshots for all permutations (`npm test -- -u`).
- [x] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [x] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [x] Made sure that all accessibility errors are resolved.
- [x] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.

---------
**Outline your feature or bug-fix below**

Updated all dependencies to be latest.
Removed lodash because it's not really needed for this project.

### CODE REVIEW

File: utils/get.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const get = (obj: any, path: string, defaultValue: any) => path.split('.')
+  .reduce((a, c) => (a && a[c] ? a[c] : (defaultValue || null)), obj);
```
Comment (zero0Halo) [inline utils/get.ts]: Since the function is written so briefly it's not really self documenting. It'd be good to have a jsdoc block above this to explain what it's for, since although we may know it's replacing lodash's `get` no one else would know.

Comment (sharkparty) [reply utils/get.ts]: 👍

Comment (sharkparty) [reply utils/get.ts]: Added a spec and docs.

File: utils/get/get.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * @param obj - an object that needs to be safely traversed `{ a: { b: true } }`
+ * @param path - a string that represents the path of the object `'a.b'`
+ * @param defaultValue - a value to return if the path doesn't resolve in the object
+ */
+export const get = (obj: any, path: string, defaultValue?: any) => path.split('.')
```
Comment (sharkparty) [inline utils/get/get.ts:7]: @zero0Halo bc I unit tested this, I noticed that I omitted the default being an optional prop. thanks for the push!

Comment (zero0Halo) [reply utils/get/get.ts:7]: One last change. Put a a description as to what the function is for on line 2. I am a comment nazi. Plain english, ex: "Our version of lodash's get() method, just less bloated" or something to that effect.

Comment (sharkparty) [reply utils/get/get.ts:7]: Nice catch.

Comment (zero0Halo) [reply utils/get/get.ts:7]: Looks good!

**Review by zero0Halo — APPROVED** [review]
Looks good.
